graphql-database-manager-dynamo/src/main/java/com/fleetpin/graphql/database/manager/dynamo/HistoryUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.fleetpin.graphql.database.manager.dynamo;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.Record;
+
+public class HistoryUtil {
+
+	public static Stream<HashMap<String, AttributeValue>> toHistoryValue(List<Record> records) {
```

Comment:
Once thing we have missed is we want to be able to control what items have history.

I think in order to get history you should have to add an annotation 
`@History`
on the classes that extend table.
Then when we do a db put we see if the type has this annotation and if we do we add a flag to the object
history = true
that this then only will process the rows that we want to capture the history on.
We should also do the check when we go to query the type make sure its a type that has the `@History` annotation and error the query if we don't have it